src/Fbx2Raw.cpp

```diff
@@ -1423,7 +1424,7 @@ bool LoadFBXFile(RawModel &raw, const char *fbxFileName, const char *textureExte
         FbxSystemUnit::m.ConvertScene(pScene);
     }
 
-    ReadNodeHierarchy(raw, pScene, pScene->GetRootNode(), "", "");
+    ReadNodeHierarchy(raw, pScene, pScene->GetRootNode(), 0, "");
```

Comment:
I guess we can trust that node->GetUniqueId() never returns zero. :)

Reply:
I cannot find nothing in the FBX docs, but in my tests the returning Id is always greather than 0 :)